superset/assets/src/visualizations/nvd3_vis.js

```diff
@@ -363,6 +363,13 @@ export default function nvd3Vis(slice, payload) {
         throw new Error('Unrecognized visualization for nvd3' + vizType);
     }
 
+    if (isTruthy(fd.show_brush) && isTruthy(fd.send_time_range)) {
```

Comment:
NIT: I've seen `!!foo` used in places as an `isTruthy` not sure if there are pros/cons but we should aim at consistency throughout the codebase. Not sure if eslint rules may exist for this. @williaster what you think?